.github/workflows/tox.yml

```diff
@@ -18,6 +18,15 @@ jobs:
       - uses: actions/setup-python@v2
         with:
           python-version: ${{ matrix.python-version }}
+      - name: Cache multiple paths
+        uses: actions/cache@v2
+        with:
+          path: |
```

Comment:
We don't need to cache `/opt/hostedtoolcache/Python/*` and `c:\hostedtoolcache\windows\python\*`

Reply:
This already redefined with new commit(s), see new version of the file

Reply:
I was referring to  `$RUNNER_TOOL_CACHE/Python/*`. Curious, could you explain why we would need to cache this directory? I can understand caching **pip cache directories** since we are installing tox.

Reply:
Installed packages are stored under lower folders in `$RUNNER_TOOL_CACHE/Python/*` - by doing this, we don't need to install those packages. `pip cache directories` may not be enough, from the logs I noticed, some packages still escape - not sure how or why. Hence my comment https://github.com/nbQA-dev/nbQA/pull/322#issuecomment-710051871

I'm now going to peek into the logs and also compare it with original builds to see if caching has helped improve speed even if its by a little bit

Reply:
Got it. I raised this concern because, if we cache `$RUNNER_TOOL_CACHE/Python/*` we might cache a lot of files. I am fine with it now. I checked the cache size on windows and ubuntu with python 3.7, the cache size is 18MB and 30MB respectively.